upi/vsphere/upi-destroy.ps1

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/pwsh
+
+. .\variables.ps1
+
+$ErrorActionPreference = "Stop"
+
+# since we do not have ca for vsphere certs, we'll just set insecure
+Set-PowerCLIConfiguration -InvalidCertificateAction:Ignore -Confirm:$false | Out-Null
```

Comment:
Maybe this is a follow up PR but we have the certificates available in the ipi destroy.

Reply:
Hmm, I never did look into making sure pwsh could see the certs.  I did this so we can make progress when install was not liking talking to vcenter.  I can make a card for this to add next if you are ok with that.

Reply:
yep sounds good